mu-index -q dies immediately with segfault
this patch fixes it
patch by Michael Guntsche <mike@it-loops.com>
---
 index/mu-index.c |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

Index: maildir-utils/index/mu-index.c
===================================================================
--- maildir-utils.orig/index/mu-index.c	2009-10-30 02:36:24.000000000 +0900
+++ maildir-utils/index/mu-index.c	2009-10-30 02:36:34.000000000 +0900
@@ -195,8 +195,9 @@
 	result = run_callback_maybe (data);
 	if (result != MU_OK)
 		return result;
-	
-	++data->_stats->_processed;
+  
+	if (!mu_index_app_quiet())
+		++data->_stats->_processed;
 		
 	/* see if we need to update/insert anything...*/
 	dbstamp = mu_storage_sqlite_message_timestamp (data->_sqlite, fullpath);
